saleor/plugins/base_plugin.py

```diff
@@ -468,9 +468,6 @@ def __str__(self):
         Tuple[Union["User", NoneType], dict],
     ]
 
-    #  Triggered when ShopFetchTaxRates mutation is called.
-    fetch_taxes_data: Callable[[Any], Any]
-
```

Comment:
Should those changes be described in the changelog as breaking changes?

Reply:
Yes, it will be added to changelog later when merging the feature branch to `main`.